docs/user/introduction.asciidoc

```diff
@@ -1,12 +1,160 @@
 [[introduction]]
-== Introduction
+== {kib} &mdash; your window into the Elastic Stack
+++++
+<titleabbrev>What is Kibana?</titleabbrev>
+++++
 
-Kibana is an open source analytics and visualization platform designed to work with Elasticsearch. You use Kibana to
-search, view, and interact with data stored in Elasticsearch indices. You can easily perform advanced data analysis
-and visualize your data in a variety of charts, tables, and maps.
+**_Explore and visualize your data and manage all things Elastic Stack._**
 
-Kibana makes it easy to understand large volumes of data. Its simple, browser-based interface enables you to quickly
-create and share dynamic dashboards that display changes to Elasticsearch queries in real time.
+Whether you’re a user or admin, {kib} makes your data actionable by providing
+three key functions:
 
-Setting up Kibana is a snap. You can install Kibana and start exploring your Elasticsearch indices in minutes -- no
-code, no additional infrastructure required.
+* **{kib} is an open-source analytics and visualization platform.**
+Use {kib} to explore your {es} data, and then build beautiful visualizations and dashboards.
+
+* **{kib} is a UI for managing {es} clusters.**
+Manage your security settings, assign user roles, monitor the Stack,
+ingest and roll up your data...the list goes on.
+
+* **{kib} is a centralized hub for our solutions.** &mdash; From log analytics to
+document discovery to SIEM, {kib} is the portal for accessing these and other capabilities.
+
+[role="screenshot"]
+image::images/intro-kibana.png[]
+
+[float]
+[[where-to-start-kibana]]
+=== Where do I start?
```

Comment:
I don't think this section belongs in the introduction. I think users want to know what they are getting into before they learn about jumping in and setting up. This section is a better fit for the getting started section.

Reply:
i moved this section to the end.